memorywipe.py

```diff
@@ -0,0 +1,496 @@
+                strongp = click.prompt("So set a random password with special characters: ", hide_input=True, confirmation_prompt=True)
+                out = subprocess.run(["sudo", "veracrypt", "-tc", "--volume-type=normal", 
+                                f"{self.partition}", "--encryption=aes", "--hash=sha-512", 
+                                "--filesystem=ntfs", "-p", f"{strongp}", "--pim=0", 
+                                "-k", '""', "--random-source=/dev/urandom"], capture_output=True)
+                click.secho(f"Finished encrypting {self.partition}\n", fg="green")
+                              
+    def _wipe_disk(self):
+        click.echo(f"Wiping {self.partition}...")
+        if not self._check_mount():
+            click.secho("Make sure the disk is unmounted!", fg="red")
+            raise click.Abort
+        else:
+            subprocess.run(["sudo", "dd", "if=/dev/random", f"of={self.partition}", "bs=1M", "status=progress"])
+            subprocess.run(["sudo", "dd", "if=/dev/zero", f"of={self.partition}", "bs=1M", "status=progress"])
+            
+            click.secho(f"{self.partition} overwritten with one write each of random data and zeroes", fg="bright_white")
+            self.name = click.prompt("Set device name")
+            subprocess.run("sudo", "mkfs.ntfs", "-L", f"{self.name}", f"{self.partition}")
+            click.secho(f"Wiped {self.partition}\n", fg="green")
+            
+    def _mount_disk(self):
+        #read -p "Set mounting folder name: " name
+        click.echo(f"Mounting {self.partition} at /media/{self.name}")
+        subprocess.run(["sudo", "mkdir", f"/media/{self.name}"])
+    
+        subprocess.run(["sudo", "mount", f"{self.partition}", f"/media/{self.name}/"])
+        click.echo("Finished mounting!\n", fg="green") 
+        del self.name
+                     
+    def _check_mount(self):
+        try:
+            click.secho("Unmounting the partition...", fg="blue")
+            # Run the grep command to check if the partition is in /proc/mounts
+            try:         
+                subprocess.run(["grep", "-qs", self.partition, "/proc/mounts"], check=True)
+                # If the above command doesn't raise a CalledProcessError, the partition is mounted
+                # Unmount the partition
+            except subprocess.CalledProcessError:
+                click.secho(f"{self.partition} is already dismounted or doesn't exist")
+                return True
+            subprocess.run(["sudo", "umount", self.partition], check=True)
+            # Check if the partition is still mounted
+            subprocess.run(["grep", "-qs", self.partition, "/proc/mounts"], check=True)
+            click.echo(f"Error: Failed to unmount {self.partition}. Please unmount manually.", fg="red")
+            return False
+        except subprocess.CalledProcessError:
+            click.secho(f"{self.partition} has been succesfully mounted", fg="green")
```

Comment:
I believe it should be "successfully unmounted" in this line?

Reply:
Apologies for the typo. It was definitely "successfully unmounted".